.github/workflows/maven.yml

```diff
@@ -0,0 +1,68 @@
+name: Arquillian-Cube Maven Build
+
+on:
+  push:
+    branches:
+      - '**'
```

Comment:
Shouldn't we trigger just on `master`?

Reply:
done

Reply:
I am thinking if we need to create another branch(main branch?) to put all new updates, changes there.

Reply:
+1, we should rename to `main` but I think we can handle that issue separately, and maybe more in general.